src/org/zaproxy/zap/extension/ascanrulesAlpha/resources/Messages.properties

```diff
@@ -3,28 +3,6 @@
 # This file defines the default (English) variants of all of the internationalised messages
 ascanalpha.desc = Active Scan Rules - alpha
 
-ascanalpha.examplefile.name=An example active scan rule which loads data from a file
```

Comment:
Well, it's easy to split those changes into two commits with git. It allows to select hunks of changes to include/exclude in/from a given commit.
If you want I can help with that.